helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -47,6 +47,30 @@ spec:
       serviceAccountName: {{ .Values.controller.name }}
       priorityClassName: system-cluster-critical
       terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
+      initContainers:
+      # Overall performance improvements
+      # See https://github.com/kubernetes/ingress-nginx/issues/1939
+      - name: {{ .Values.initContainer.name }}
+        command:
+        - sh
+        - -c
+        - |
+          set -o errexit
+          set -o xtrace
+          until sudo sysctl -w net.core.somaxconn=32768
+          do
+            sleep 60s
+          done
+          until sudo sysctl -w net.ipv4.ip_local_port_range="1024 65535"
+          do
+            sleep 60s
+          done
```

Comment:
```suggestion
          set -o errexit ; set -o xtrace ; set -o nounset
          
          sudo sysctl -w net.core.somaxconn=32768
```

Reply:
Since we're optimizing already, dropped also setting of `net.ipv4.ip_local_port_range` from here, it can be and we should set via securityContext sysctls since it's safe (compared to `net.core.somaxconn` which is currently considered unsafe) https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#setting-sysctls-for-a-pod